test/integration/TicketFrontRun.js

```diff
@@ -87,168 +84,163 @@ describe("TicketFrontRun", ([deployer, broadcaster, evilSybilAccount, evilNonAct
         reserve = 100000
         reserveAlloc = reserve / (await bondingManager.getTranscoderPoolSize()).toNumber()
 
-        await broker.fundDepositAndReserve(
-            deposit,
-            reserve,
-            {from: broadcaster, value: deposit + reserve}
-        )
+        await broker.connect(broadcaster).fundDepositAndReserve(deposit, reserve, {value: deposit + reserve})
 
-        const roundLength = await roundsManager.roundLength.call()
-        await roundsManager.mineBlocks(roundLength.toNumber() * 1000)
-        await roundsManager.setBlockHash(web3.utils.keccak256("foo"))
+        const roundLength = await roundsManager.roundLength()
+        await roundsManager.mineBlocks(roundLength.mul(1000))
+        await roundsManager.setBlockHash(ethers.utils.solidityKeccak256(["string"], ["foo"]))
         await roundsManager.initializeRound()
     })
 
-    beforeEach(async () => {
-        snapshotId = await rpc.snapshot()
-    })
-
-    afterEach(async () => {
-        await rpc.revert(snapshotId)
-    })
-
```

Comment:
We should keep this. This creates a snapshot of the chain using `evm_snapshot` before each test and after each test resets the state to that snapshot. 

`snapshotId = await rpc.snapshot()` could also be moved to the bottom of the `before` hook